.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+  "singleQuote": true,
+  "bracketSpacing": true,
+  "trailingComma": "none"
+}
```

Comment:
Trailing EOL should always be there; at the very least, missing EOLs should be consistent across the project. (After all, isn't this PR about linting?)